rocAL/rocAL_pybind/amd/rocal/fn.py

```diff
@@ -541,7 +541,7 @@ def random_bbox_crop(*inputs,all_boxes_above_threshold = True, allow_no_crop =Tr
     aspect_ratio = b.CreateFloatUniformRand(aspect_ratio[0], aspect_ratio[1])
 
     # pybind call arguments
-    kwargs_pybind = {"all_boxes_above_threshold":all_boxes_above_threshold, "no_crop": allow_no_crop, "p_aspect_ratio":aspect_ratio, "has_shape":has_shape, "crop_width":crop_width, "crop_height":crop_height, "num_attemps":num_attempts, "p_scaling":scaling, "total_num_attempts":total_num_attempts }
+    kwargs_pybind = {"all_boxes_above_threshold":all_boxes_above_threshold, "no_crop": allow_no_crop, "p_aspect_ratio":aspect_ratio, "has_shape":has_shape, "crop_width":crop_width, "crop_height":crop_height, "num_attemps":num_attempts, "p_scaling":scaling, "total_num_attempts":total_num_attempts, "seed":seed }
```

Comment:
> General Comments:
> 
>     1. Previously there was augmentation count and all outputs will be of same width and height but in this pipeline that will not be the case we need to handle this in a different way
> 
>     2. Please check with @shobana-mcw  and remove IOU matcher related changes in this PR
> 
>     3. Please sync your branches with develop since the pybind changes are made and we would need to work with that



Have addressed the point 2 and 3.

For point 1 - This is a basic support given. 
We need to support different augmentations with different data-types & shapes in the iterator. 
Please note @shobana-mcw

Reply:
Will we issue this in future PR?

Reply:
WIll be issued in future PR